sdk/storage/azure-storage-blob/tests/test_block_blob.py

```diff
@@ -76,6 +78,26 @@ def _create_source_blob(self, data):
         blob_client.upload_blob(data, overwrite=True)
         return blob_client
 
+    def _get_bearer_token_string(
+        self, prefix: str = "Bearer ",
```

Comment:
A Bearer token is always going to have a the `Bearer` prefix so it probably doesn't need to be a parameter.